### PR TITLE
[WIP] Turn editor fontsize customizable

### PR DESCRIPTION
**Pull request by lenhard**
Partial fix for #3034

The components in the entry edtor now use the menu font size that can be set in the preferences.

Missing:
 - The text field of the Crossref editor
 - The recommendations by the auto completion
 - The tooltip text for the buttons

- [x] Change in CHANGELOG.md described
- [ ] Tests created for changes
- [ ] Screenshots added (for bigger UI changes)
- [x] Manually tested changed features in running JabRef
- [ ] Check documentation status (Issue created for outdated help page at [help.jabref.org](https://github.com/JabRef/help.jabref.org/issues)?)
- [ ] If you changed the localization: Did you run `gradle localizationUpdate`?

### CODE REVIEW

**Comment by lenhard** [comment]
Thanks for the tip, I will try it out! :)

**Comment by Siedlerchr** [comment]
See also this article for an inspiration: https://fxdocs.github.io/docs/index.html#_styleable_properties

**Comment by lenhard** [comment]
The style inheritance works for some items, but not for all (such as context menu items). 

Some of the buttons are defined in FXML and their tooltips are defined in FXML. @Siedlerchr @tobiasdiez Any tipps on how to make the font size customizable for these items? Moreover, how does this work with localization?

**Comment by tobiasdiez** [comment]
You should be able to style the context menu (and similar the tooltip) by using appropiate css tags, e.g.
````css
.context-menu {
 -fx-font-size: Globals.prefs.getInt(JabRefPreferences.MENU_FONT_SIZE) + "pt;");
}
````
See https://stackoverflow.com/questions/11127999/how-do-you-set-the-style-for-a-javafx-contextmenu-using-css.

File: src/main/java/org/jabref/gui/entryeditor/FieldsEditorTab.java [hunk 1]
```diff
@@ -179,7 +180,8 @@ private Region setupPanel(JabRefFrame frame, BasePanel bPanel, boolean compresse
             gridPane.setStyle(
                     "text-area-background: " + convertToHex(GUIGlobals.validFieldBackgroundColor) + ";"
                             + "text-area-foreground: " + convertToHex(GUIGlobals.editorTextColor) + ";"
-                            + "text-area-highlight: " + convertToHex(GUIGlobals.activeBackgroundColor) + ";");
+                            + "text-area-highlight: " + convertToHex(GUIGlobals.activeBackgroundColor) + ";"
+            + "text-area-font-size: " + Globals.prefs.getInt(JabRefPreferences.MENU_FONT_SIZE) + "pt;");
```
Comment (tobiasdiez) [inline src/main/java/org/jabref/gui/entryeditor/FieldsEditorTab.java]: adding `+ "-fx-font-size: 20pt;"` here without any other code changes seems to do the trick.

**Comment by lenhard** [comment]
#3034 supersedes this PR, therefore I am closing it.

**Pull request closed without merging**